src/onboarding/registration/EnableBiometry.tsx

```diff
@@ -145,6 +166,10 @@ const styles = StyleSheet.create({
     paddingHorizontal: 40,
     alignItems: 'center',
   },
+  containerLeftAligned: {
+    paddingTop: 186,
+    paddingHorizontal: 40,
```

Comment:
These paddings seem a bit suspect. Is there a way to get the alignment you want without these static values? Maybe a flex with align / justify content?

Reply:
> These paddings seem a bit suspect. Is there a way to get the alignment you want without these static values? Maybe a flex with align / justify content?

Definitely there is; these numbers were not my original so I hesitate touching them, however let me get a quick patch in.